<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDlg</class>
 <widget class="QDialog" name="ConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Media Library</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>save path:</string>
           </property>
           <property name="buddy">
            <cstring>savePathLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="savePathLineEdit">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectSavePathPushButton">
           <property name="text">
            <string>select Path...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QListWidget" name="directoriesListWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="addDirectoryPushButton">
             <property name="text">
              <string>add directory...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editDirectoryPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>edit directory...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeDirectoryPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>remove directory</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>re-scanning</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>auto re-scan interval</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="suffix">
             <string> hours</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>rescan now</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Playlists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Format Template:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="formatTemplateLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="placeholderArtistButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Artist </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="placeholderAlbumButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Album</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="placeholderTitleButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="placeholderTracknumberButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Tracknumber</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="placeholderDurationButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Duration</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="placeholderFilePathButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Filepath</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="placeholderFileNameButton">
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>248</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectSavePathPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>selectSavePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDirectoryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>selectDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDirectoryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>editDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeDirectoryPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDlg</receiver>
   <slot>removeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>directoriesListWidget</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>ConfigDlg</receiver>
   <slot>updateButtonActivity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectSavePath()</slot>
  <slot>selectDirectory()</slot>
  <slot>editDirectory()</slot>
  <slot>removeDirectory()</slot>
  <slot>updateButtonActivity()</slot>
 </slots>
</ui>
